src/properties/property-accessors.html

```diff
@@ -27,6 +29,19 @@
     proto = Object.getPrototypeOf(proto);
   }
 
+  /**
+   * Used to save the value of a property that will be overridden with
```

Comment:
This is a `@private` function in the closure, not accessible outside this file. Right now I'm erring on the side of documenting/annotating everything in prep for making Polymer buildable with jsCompiler.